sample/src/main/java/com/karumi/dexter/sample/SampleActivity.java

```diff
@@ -144,11 +145,6 @@ private void createPermissionListeners() {
                 R.string.all_permissions_denied_feedback)
                 .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
                 .build());
-    cameraPermissionListener = new CompositePermissionListener(feedbackViewPermissionListener,
-        SnackbarOnDeniedPermissionListener.Builder.with(rootView,
-            R.string.camera_permission_denied_feedback)
-            .withOpenSettingsButton(R.string.permission_rationale_settings_button_text)
-            .build());
```

Comment:
Sorry, I didn't read it. You've removed all the code needed to show the rationale and open settings button and replaced this code with a ``cameraPermissionListener = new SampleBackgroundThreadPermissionListener(this);``. We need to keep the old example behaviour.